src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 2;
+  const SHORT_TERM_DISCOUNT = 20;
+  const PRICE_PER_DAY = 40;
+
+  let sum = days * PRICE_PER_DAY;
+
+  if (days < LONG_TERM && days > SHORT_TERM) {
```

Comment:
The condition `days > SHORT_TERM` will not work as expected because it should be `days >= SHORT_TERM` to include the third day as the starting point for the short-term discount.